test/Test/Pos/BinarySpec.hs

```diff
@@ -58,40 +58,42 @@ tinyVarIntSpec = describe "TinyVarInt" $ do
                 B.biSize (B.TinyVarInt n) `shouldBe` b
         maxnum = 2^(14::Int)-1
 
-    describe "1 byte" $ do
```

Comment:
I will review places where TinyVarInt is used and tell you. It should be okay in most places, I think, but better check to be sure.